SQL/0000-00-00-schema.sql

```diff
@@ -762,6 +802,33 @@ CREATE TABLE `mri_protocol_checks` (
     FOREIGN KEY (`Scan_type`) REFERENCES `mri_scan_type` (`ID`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
+CREATE TABLE `mri_protocol_checks_group` (
+    `MriProtocolChecksGroupID`   INT(2) UNSIGNED NOT NULL AUTO_INCREMENT,
+    `Name`                       VARCHAR(255)    NOT NULL,
+    PRIMARY KEY (`MriProtocolChecksGroupID`)
+) ENGINE = InnoDB  DEFAULT CHARSET=utf8;
+
+INSERT INTO `mri_protocol_checks_group` VALUES(1, 'Default MRI protocol checks group');
```

Comment:
The `1` should be generated by the auto_increment. And, the insert at line 830 should use a subselect WHERE Name = 'Default MRI protocol checks group' to get this ID

Reply:
Ok